client/services/useApplicationDetails.ts

```diff
@@ -143,6 +141,20 @@ export const useApplicationDetails = (id: number | number[] | undefined) => {
           onClick: () => updateStatus(id, ApplicationStatus.DENIED),
```

Comment:
I'd remove `onClick: () => updateStatus(id, ApplicationStatus.DENIED),` for `RECEIVED` and `ASSIGNED`. Only admin can deny application and only when it's in `WORKSHOP` status